transform.js

```diff
@@ -54,19 +54,25 @@ function transform (filename, options) {
       return
     }
 
-    try {
-      const tmpAst = falafel(src, { ecmaVersion: 8 }, identifyModuleName)
-      ast = falafel(tmpAst.toString(), { ecmaVersion: 8 }, extractNodes)
-    } catch (err) {
-      return self.emit('error', err)
-    }
-
-    // transform all detected nodes and
-    // close stream when done
-    mapLimit(nodes, Infinity, iterate, function (err) {
+    findTransforms(filename, function (err, transforms) {
       if (err) return self.emit('error', err)
-      self.push(ast.toString())
-      self.push(null)
+
+      opts.transform = transforms.concat(opts.transform || []).concat(opts.t || [])
```

Comment:
Ehhh, I think this is wrong. Maybe if there are multiple files it will fail? Let me write a test.

Reply:
Ah, no, that's definitely wrong. This is only called once per file, because this is in the end handler. My bad.